llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -730,6 +742,9 @@ class Instruction : public sandboxir::User {
   friend class CastInst;           // For getTopmostLLVMInstruction().
   friend class PHINode;            // For getTopmostLLVMInstruction().
   friend class UnreachableInst;    // For getTopmostLLVMInstruction().
+  friend class CmpInst;            // For getTopmostLLVMInstruction().
+  friend class ICmpInst;           // For getTopmostLLVMInstruction().
```

Comment:
Since `create()` is in `CmpInst` my guess is that there won't be a need for `getTopmostLLVMInstruction()` in `ICmpInst` and `FCmpInst`. But I may be wrong.